src/test/java/core/basesyntax/ReadParserTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.ReadParser;
+import core.basesyntax.service.impl.ReadParserImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReadParserTest {
+    private ReadParser readParser;
+
+    @Before
+    public void setUp() {
+        readParser = new ReadParserImpl();
+    }
+
+    @Test
+    public void readParse_ValidData_Ok() {
+        List<String> lines = new ArrayList<>();
+        lines.add("type,fruit,quantity");
+        lines.add("b,apple,10");
+        lines.add("p,banana,5");
+        List<FruitTransaction> transactions = readParser.parse(lines);
+        assertNotNull(transactions);
+        assertEquals(2, transactions.size());
```

Comment:
It is not unit test, you testing that there are 2 transactions and that values you provided correct